web/cobrands/fixmystreet.com/base.scss

```diff
@@ -299,3 +299,57 @@ html.lazyload .js-lazyload {
     margin-bottom: 0;
   }
 }
+
+.postcode-form-test {
+    margin: 0 -1em;
+    padding: 1em;
+    background-color: $primary;
+
+    label {
+        margin: 0;
+    }
+
+    #geolocate_link {
+        background: transparent !important;
```

Comment:
Using `!important`, should be a comment explaining why.

Reply:
Done!